doc/man/bitcoind.1

```diff
@@ -456,6 +456,9 @@ Set maximum size of high\-priority/low\-fee transactions in bytes
 .IP
 Set lowest fee rate (in BTC/kB) for transactions to be included in block
 creation. (default: 0.00001)
+\fB\-bip148\fR
```

Comment:
Should be "-reorgdefencebip148"
Core has the responsibility of providing users with the safest software. If BIP148 has a risk of wiping out blocks accepted by core, then it is also possible for miners to maliciously steer core nodes away from BIP148 for 1 or 2 blocks (in the case where the legacy chain has been reorged onto BIP148). At this point it is irresponsible for core to not have a solution ready "please enable -reorgdefencebip148 flag" and instead only have the option of scrambling to write, test, and release an emergency patch and say "users please download this emergency release".

Hopefully this puts the bip148 in core discussion into perspective.